diff --git a/Boss/Mod/AmountSettingsHandler.cpp b/Boss/Mod/AmountSettingsHandler.cpp
index c7eaae9..0677624 100644
--- a/Boss/Mod/AmountSettingsHandler.cpp
+++ b/Boss/Mod/AmountSettingsHandler.cpp
@@ -15,23 +15,23 @@
 namespace {
 
 /* Default channel boundaries.  */
-auto const default_min_channel = Ln::Amount::sat(  500000);
-auto const default_max_channel = Ln::Amount::sat(16777215);
+auto const default_min_channel = Ln::Amount::sat(   500000);
+auto const default_max_channel = Ln::Amount::sat(100000000);
 
 /* Default amount to always leave onchain for future
  * channel management actions.  */
-auto const default_reserve =     Ln::Amount::sat(   30000);
+auto const default_reserve =     Ln::Amount::sat(   100000);
 
 /* The absolute lowest min_channel setting.  */
-auto const min_min_channel =     Ln::Amount::sat(  500000);
+auto const min_min_channel =     Ln::Amount::sat(   500000);
 /* How much larger the max_channel should be over the min_channel.  */
-auto const max_channel_factor = double(2.0);
+auto const max_channel_factor = double(5.0);
 /* How much larger the channel-creation trigger should be over
  * the min_channel.  */
 auto const trigger_factor = double(2.0);
 /* How much to add to the channel-creation trigger above, to get
  * the amount to leave after creation.  */
-auto const additional_remaining = Ln::Amount::sat(20000);
+auto const additional_remaining = Ln::Amount::sat(50000);
 
 Ln::Amount parse_sats(Jsmn::Object value) {
 	auto is = std::istringstream(std::string(value));
diff --git a/Boss/Mod/ChannelCreationDecider.cpp b/Boss/Mod/ChannelCreationDecider.cpp
index fdb3921..df32379 100644
--- a/Boss/Mod/ChannelCreationDecider.cpp
+++ b/Boss/Mod/ChannelCreationDecider.cpp
@@ -27,16 +27,16 @@ auto const onchain_percent_min = double(10);
  * total funds, create channels.
  * If below this, wait for low-fee region.
  */
-auto const onchain_percent_max = double(25);
+auto const onchain_percent_max = double(33);
 
 /* If onchain funds are above or equal to this amount, but in
  * onchain-percent terms is below onchain_percent_min, create
  * channels anyway --- the channels are going to be fairly
  * large still, so okay even if below onchain_percent_min.
  * This only comes into play for really rich nodes with
- * 1.6777215 BTC or more.
+ * 10 BTC or more.
  */
-auto const max_onchain_holdoff = Ln::Amount::sat(16777215);
+auto const max_onchain_holdoff = Ln::Amount::sat(100000000);
 
 }
 
@@ -178,7 +178,7 @@ private:
 					      , comment
 					      ]() {
 			return Boss::log( bus, Info
-					, "ChannelCreationDecider: %s."
+					, "ChannelCreationDecider: %s. "
 					  "Will create channels worth %s "
 					  "(%s reserved for onchain actions)."
 					, comment.c_str()
diff --git a/CHANGELOG.md b/CHANGELOG.md
index b9aa30c..13c7355 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Changed
+
+- `ChannelCreationDecider` now holds off on creating channels if the onchain amount is small relative to all your funds and is small for a "large" channel (1 BTC).
+- Default maximum channel size is now 1 BTC.
+- Increment of default reserve to 100000 satoshi.
+
 ## [0.14.1] - 2024-12-05: "Hand at the Grindstone"
 
 ### Added
diff --git a/README.md b/README.md
index 9fbb3ea..65d19e5 100644
--- a/README.md
+++ b/README.md
@@ -466,8 +466,8 @@ will make.
 
 The defaults are:
 
-* Minimum: 500000sats = 5mBTC
-* Maximum: 16777215sats = 167.77215mBTC
+* Minimum:    500000sats = 5mBTC
+* Maximum: 100000000sats = 1000mBTC
 
 Specify the value in satoshis without adding any unit
 suffix, e.g.
